lib/process.js

```diff
@@ -181,13 +181,18 @@ function processOperations(api, parentPath, topLevel, options) {
 function processTransactions(api, parentOp, parentPath, topLevel, options) {
     var transactions = []
 
+    // Replace newlines with a space
+    var stripNewlines = function(str) {
```

Comment:
Let's just put it at the bottom of this file with the other local helpers.

Thanks 😄 !